### PR TITLE
Implement try catch strategy Security

### LINKED ISSUE #3394: Refactor try catch on Security section
| Parent Issue |
|:---:|
|#3265| 

**Description**
In order to improve better error handling in the front-end, we should implement a better try-catch strategy in the Security section, services, and functions.

### PR DESCRIPTION
**Pull request by pablomarga**
Hi team,
This PR implement our new error handling strategy on Security section

Close #3394

### CODE REVIEW

File: public/components/security/policies/create-policy.tsx [hunk 1]
```diff
@@ -182,7 +187,18 @@ export const CreatePolicyFlyout = ({ closeFlyout }) => {
...
+        severity: UI_ERROR_SEVERITIES.BUSINESS,
+        store: true,
+        error: {
+          error: error,
+          message: error.message || error,
+          title: error.name || error,
+        },
+      };
+      getErrorOrchestrator().handleError(options);
       return;
```
Comment (gabiwassan) [inline public/components/security/policies/create-policy.tsx:202]: This return is not necessary

File: public/components/security/policies/policies-table.tsx [hunk 2]
```diff
@@ -99,7 +103,18 @@ export const PoliciesTable = ({policies, loading, editPolicy, createPolicy,updat
                     ErrorHandler.info('Policy was successfully deleted');
                     await updatePolicies();
                 }catch(error){
-                    ErrorHandler.handle(error, 'Error deleting policy');
+                    const options = {
```
Comment (gabiwassan) [inline public/components/security/policies/policies-table.tsx:106]: Check indentation here (prettier 🤓)

**Review by gabiwassan — CHANGES_REQUESTED** [review]
Great job!! just some details. 💪🏼

File: public/components/security/roles/roles-table.tsx [hunk 3]
```diff
@@ -1,153 +1,180 @@
...
-                onConfirm={async () => {
-                    try{
-                        const response = await WzRequest.apiReq(
-                        'DELETE',
-                        `/security/roles/`,
-                        {
-                            params: {
-                                role_ids: item.id
-                            }
+  const columns = [
```
Comment (gabiwassan) [inline public/components/security/roles/roles-table.tsx:31]: beautiful 😍

**Review by gabiwassan — APPROVED** [review]
LGTM ! 🚀

File: public/components/security/main.tsx [hunk 4]
```diff
@@ -56,49 +66,60 @@ export const WzSecurity = compose(
...
-      const ApiCheck = await GenericRequest.request('POST',
-        '/api/check-api',
-        currentApi
-      );
+      const ApiCheck = await GenericRequest.request('POST', '/api/check-api', currentApi);
       return ApiCheck.data.allow_run_as;
-
     } catch (error) {
-      ErrorHandler.handle(error, 'Error checking the current API');
+      throw new Error(error);
```
Comment (Machi3mfl) [inline public/components/security/main.tsx:85]: I think is better only `throw` the error, without the `new`. Except, when set a custom message error.

**Review by Machi3mfl — APPROVED** [review]

**Pull request merged**